platform/entanglement/src/actions/MoveAction.js

```diff
@@ -40,7 +40,18 @@ define(
         }
 
         MoveAction.prototype = Object.create(AbstractComposeAction.prototype);
-        MoveAction.appliesTo = AbstractComposeAction.appliesTo;
+
+        MoveAction.appliesTo = function (context) {
+            var applicableObject =
+                context.selectedObject || context.domainObject;
+
+            if (applicableObject && applicableObject.model.locked) {
+                return false;
+            }
+
+            return !!(applicableObject &&
+                applicableObject.hasCapability('context'));
```

Comment:
this effectively does the same thing?

Reply:
yes, I think this might throw error later due to changes in PR: New eslint rules auto fix #3058

here is change for similar statement:
https://github.com/nasa/openmct/pull/3058/files?file-filters%5B%5D=.js#diff-2beaaaf58a5327959536f8894652c627R82

Reply:
good catch! I forgot about the new lint changes